<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKUdpClientAdvanceSettingWidget</class>
 <widget class="QWidget" name="SAKUdpClientAdvanceSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDP Advance Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="0" colspan="4">
    <widget class="QLabel" name="label_9">
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:green
}</string>
     </property>
     <property name="text">
      <string>Multicast settings</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:green
}</string>
     </property>
     <property name="text">
      <string>Unicast settings</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame{
border:none
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="3">
       <widget class="QPushButton" name="deletePushButton">
        <property name="text">
         <string>DeleteMulticast</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="addPushButton">
        <property name="text">
         <string>AddMulticast</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="cancelPushButton">
        <property name="text">
         <string>Cancle</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="multicastCheckBox">
     <property name="text">
      <string>Enable multicast</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:green
}</string>
     </property>
     <property name="text">
      <string>Broadcast settings</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Broadcast port</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="broadcastCheckBox">
     <property name="text">
      <string>Enable broadcast</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="broadcastPortLineEdit">
     <property name="text">
      <string notr="true">55555</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QCheckBox" name="unicastCheckBox">
     <property name="text">
      <string>Enable unicast</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SAKUdpClientAdvanceSettingWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
